### PR TITLE
[plugin.video.opentakserver] 1.0.0

### PR DESCRIPTION
**Pull request by brian7704**
### Description
View video streams and recordings from [OpenTAKServer](https://github.com/brian7704/OpenTAKServer)
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

**Comment by basrieter** [comment]
Please make the add-on checker happy.

**Comment by brian7704** [comment]
I resized the logo and removed the transparency. Also renamed main.py to addon.py due to the 15 line warning for main.py

File: plugin.video.opentakserver/addon.py [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+    f = xbmcvfs.File(os.path.join(__addondir__, "auth_token"), 'r')
+    auth_token = f.read()
+    f.close()
+    return auth_token
+
+
+def get(url):
+    auth_token = get_auth_token_from_file()
+
+    r = requests.get(url, params={'auth_token': auth_token}, verify=False)
```
Comment (basrieter) [inline plugin.video.opentakserver/addon.py]: I think `verify=False` should be configurable: some might actually have certificates and then they don't want to just ignore invalid ones?

File: plugin.video.opentakserver/addon.py [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    Create a URL for calling the plugin recursively from the given set of keyword arguments.
+
+    :param kwargs: "argument=value" pairs
+    :return: plugin call URL
+    :rtype: str
+    """
+    return '{}?{}'.format(URL, urlencode(kwargs))
+
+
+def list_videos():
```
Comment (basrieter) [inline plugin.video.opentakserver/addon.py]: This will always return an empty listing?

File: plugin.video.opentakserver/addon.xml
Comment (basrieter) [inline plugin.video.opentakserver/addon.xml:1]: Please provide language for the add-on using the `<language>` tag and perhaps add a `<website>` or `<source>`?

**Review by basrieter — CHANGES_REQUESTED** [review]
Please have a look at my comments.

Besides that: could you explain a bit what this plugin will do?

**Comment by brian7704** [comment]
I'll fix those issues and update the PR.

I'm the developer of [OpenTAKServer](https://github.com/brian7704/OpenTAKServer) which is a self hosted, open source backend for [ATAK](https://play.google.com/store/apps/details?id=com.atakmap.app.civ&hl=en_US). One of OpenTAKServer's features is the ability to pull in video feeds from various sources such as IP cameras, online streams, drones, etc using [MediaMTX](https://github.com/bluenviron/mediamtx). This plugin allows users to view those streams in Kodi.

File: plugin.video.opentakserver/addon.py (hunk 1 above)
Comment (brian7704) [reply plugin.video.opentakserver/addon.py]: I've updated the PR with this option

File: plugin.video.opentakserver/addon.py (hunk 2 above)
Comment (brian7704) [reply plugin.video.opentakserver/addon.py]: This has been removed

File: plugin.video.opentakserver/addon.xml
Comment (brian7704) [reply plugin.video.opentakserver/addon.xml:1]: These tags have been added

**Review by basrieter — APPROVED** [review]

**Pull request merged**